bindings/protocols/bacnet/index.html

```diff
@@ -1,1192 +1,1751 @@
+                                <li>from-time</li>
+                                <li>to-time</li>
+                                <li>recipient</li>
+                                <li>process-identifier</li>
+                                <li>issue-confirmed</li>
+                                <li>transitions</li>
+                            </ul>
+                        </td>
+                        <td>
+                            The simplifications below allow to omit the WoT Data Model.
+                            <ul>
+                                <li>valid-days &#8592; all days</li>
+                                <li>from-time &#8592; all time</li>
+                                <li>to-time &#8592; all time</li>
+                                <li>recipient &#8592; BACnet address of the WoT Consumer</li>
+                                <li>process-identifier &#8592; BACnet process ID of the WoT consumer</li>
+                                <li>issue-confirmed-notifications &#8592; always true</li>
+                                <li>transitions &#8592; all transitions</li>
+                            </ul>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>EventAffordance.cancellation</td>
+                        <td>Not modeled, cancellation is implicit</td>
+                        <td>Same as EventAffordance.subscription</td>
+                        <td>Same as EventAffordance.subscription</td>
+                    </tr>
+                    <tr>
+                        <td>EventAffordance.data</td>
+                        <td>
+                            <ul>
+                                <li>source: IRI of the BACnet object sending the event</li>
+                                <li>time: Timestamp of the event</li>
+                                <li>prio: Priority of the event</li>
+                                <li>ackReq (opt.): Flag indicating if acknowledge is required. Default is false.</li>
+                                <li>toState: The target state of the event transition</li>
+                                <li>origSource (opt.): In case of algorithmic reporting, the IRI of the object monitored by the event-sending object.</li>
+                                <li>eventType (opt.): If the event is a regular event transition, an acknowledge notification or a general sync-required event. Default is regular.</li>
+                                <li>values (opt.): Additional values associated with the event, e.g. high-limit in case the value exceeded an OffNormal limit.</li>
+                                <li>message (opt.): The message associated with the event.</li>
+                                <li>inactive (opt.): Flag if the event is not active anymore, default is false. This is needed for inactive, but not yet acknowledged events; they are there for symmetry with GetCurrentEvents action.</li>

```

Comment:
```suggestion
                                <li>source — IRI of the BACnet object sending the event</li>
                                <li>time — Timestamp of the event</li>
                                <li>prio — Priority of the event</li>
                                <li>ackReq — (optional) Flag indicating whether acknowledgement is required. Default is false.</li>
                                <li>toState: The target state of the event transition</li>
                                <li>origSource — (optional) In algorithmic reporting, the IRI of the object monitored by the event-sending object.</li>
                                <li>eventType — (optional) If the event is a regular event transition, an acknowledgement notification or a general sync-required event. Default is regular.</li>
                                <li>values — (optional) Additional values associated with the event, e.g., high-limit if value exceeded an OffNormal limit.</li>
                                <li>message — (optional) The message associated with the event.</li>
                                <li>inactive — (optional) Flag whether the event is not active anymore. Default is false. This is needed for inactive but not yet acknowledged events; they are there for symmetry with GetCurrentEvents action.</li>
```

Reply:
I kept the colon, took over the wording.